app/screens/TalkDetailsScreen/BreakDetailsScreen.tsx

```diff
@@ -21,6 +21,8 @@ import {
   getFullWidthImageDimensions,
   MIN_HEADER_HEIGHT,
   Screen,
+  SCREEN_CONTENT_WIDTH,
+  SCREEN_WIDTH,
```

Comment:
In general, my gut feeling is to have a single constants file that we can reference with all the constants for the app. How do you generally approach this?

Reply:
as discussed, will be address in a separate PR. all consts will live in `app/utils/constants`